src/test/java/ConsoleIOTest.java

```diff
@@ -1,2 +1,44 @@
+import org.junit.Test;
+
+import java.io.ByteArrayInputStream;
+import java.io.InputStream;
+
+import static org.junit.Assert.assertEquals;
+
 public class ConsoleIOTest {
+
+    ConsoleIO consoleFixString;
+    ConsoleIO consoleFixNumber;
+    ConsoleIO consoleFixBoolean;
+
+    @Test
+    public void getStringInputCallsGetInput() {
+        ConsoleIOSpy consoleIOSpy = new ConsoleIOSpy(System.in, System.out);
+        consoleIOSpy.getStringInput(1, "first name");
+        assertEquals(true, consoleIOSpy.getInputHasBeenCalled);
```

Comment:
When you add the methods in your spy, here you would be asserting on the method and not on the value of the variable.